python/ray/data/_internal/logical/interfaces/logical_operator.py

```diff
@@ -82,3 +82,7 @@ def is_lineage_serializable(self) -> bool:
         objects aren't available on the deserialized machine.
         """
         return True
+
+    @classmethod
+    def is_read_op(cls):
+        return False
```

Comment:
there is already an `is_read()`

Reply:
Yeah, and i already got confused by it once, hence renaming it (would prefer to get rid of it eventually, once we unify readers and datasources)